koku/masu/processor/tasks.py

```diff
@@ -466,29 +454,26 @@ def update_summary_tables(  # noqa: C901
             return
         worker_cache.lock_single_task(task_name, cache_args, timeout=timeout)
 
-    stmt = (
-        f"update_summary_tables called with args: "
-        f" schema_name: {schema_name}, "
-        f" provider: {provider}, "
-        f" start_date: {start_date}, "
-        f" end_date: {end_date}, "
-        f" invoice_month: {invoice_month}, "
-        f" manifest_id: {manifest_id}, "
-        f" tracing_id: {tracing_id}"
+    LOG.info(
+        log_json(
+            tracing_id,
+            "starting summary table update",
+            context,
+            start_date=start_date,
+            end_date=end_date,
+            invoice_month=invoice_month,
+        )
     )
-    LOG.info(log_json(tracing_id, stmt))
 
     try:
-        updater = ReportSummaryUpdater(schema_name, provider_uuid, manifest_id, tracing_id)
+        updater = ReportSummaryUpdater(schema, provider_uuid, manifest_id, tracing_id)
         start_date, end_date = updater.update_summary_tables(
             start_date, end_date, tracing_id, invoice_month=invoice_month
         )
         if ocp_on_cloud:
             ocp_on_cloud_infra_map = updater.get_openshift_on_cloud_infra_map(start_date, end_date, tracing_id)
     except ReportSummaryUpdaterCloudError as ex:
-        LOG.info(
-            log_json(tracing_id, f"Failed to correlate OpenShift metrics for provider: {provider_uuid}. Error: {ex}")
-        )
+        LOG.info(log_json(tracing_id, f"failed to correlate OpenShift metrics: error: {ex}", context))
```

Comment:
The str are repr of a `ReportSummaryUpdaterCloudError` exception is hopefully useful. If not, `traceback.format_exc()` will provide useful details.